statsmodels/tsa/stattools.py

```diff
@@ -1096,27 +1098,48 @@ def ccf(x, y, adjusted=True, fft=True):
     x, y : array_like
         The time series data to use in the calculation.
     adjusted : bool
-        If True, then denominators for cross-correlation is n-k, otherwise n.
+        If True, then denominators for cross-correlation are n-k, otherwise n.
     fft : bool, default True
         If True, use FFT convolution.  This method should be preferred
         for long time series.
+    nlags : int, optional
+        Number of lags to return cross-correlations for. If not provided,
+        the number of lags equals len(x).
+    alpha : float, optional
+        If a number is given, the confidence intervals for the given level are
+        returned. For instance if alpha=.05, 95 % confidence intervals are
+        returned where the standard deviation is computed according to
+        1/sqrt(len(x)).
 
     Returns
     -------
     ndarray
-        The cross-correlation function of x and y.
+        The cross-correlation function of x and y: the element at index k
+        is the correlation between {x[k], x[k+1], ..., x[n]} and {y[0], y[1], ..., y[m-k]},
+        where n and m are the lengths of x and y, respectively.
+    confint : ndarray, optional
+        Confidence intervals for the CCF at lags 0, 1, ..., nlags-1. Shape
+        (nlags, 2). Returned if alpha is not None.
```

Comment:
This should mention that it uses sqrt(nobs) as standard error for confidence interval

Reply:
Done